### ISSUE TITLE
Java client supports multi-tenancy for EvaluateDecision RPC

### ISSUE THREAD

**Issue by korthout** [camunda/zeebe#13557]

**Comment by koevskinikola** [comment]
@korthout did you plan to add anything specific to the description of this issue?

**Comment by korthout** [comment]
@koevskinikola No, I just needed an issue reference to refer to from the code (in todos).

Since none of the tasks in the umbrella issue were converted to issues yet I expected that this would come later when we'd start work on it.
